jbmc/src/java_bytecode/generic_parameter_specialization_map_keys.cpp

```diff
@@ -58,13 +58,13 @@ void generic_parameter_specialization_map_keyst::insert_pairs(
           pair.first.get_name()))
     {
       const irep_idt &key = pair.first.get_name();
-      if(generic_parameter_specialization_map.count(key) == 0)
-        generic_parameter_specialization_map.emplace(
-          key, std::vector<reference_typet>());
-      (*generic_parameter_specialization_map.find(key))
-        .second.push_back(pair.second);
+      auto map_it = generic_parameter_specialization_map
```

Comment:
:pick: This could be made `const` (the entry that the iterator points to will be mutated but the iterator itself won't be). :shell: